RELEASE.md

```diff
@@ -2,7 +2,7 @@
 
 ## Major features and improvements
 * Added the following datasets:
-  - `ParquetDaskDataSet` in `kedro.contrib.io.dask` for handling parquet datasets using Dask dataframe.
+  - `ParquetDaskDataSet` in `kedro.contrib.io.dask` for handling parquet datasets using Dask dataframes.
```

Comment:
Just a heads up, we need to call the dataset `DaskParquetDataSet` in line with a new naming convention that we're adopting following a conversation on #182.